experiments/submit.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
May be it is better to add a default values for all env variables? There is an example of such a script in repo 'https://github.com/fonhorst/LightAutoML_Spark_DevTools': scenarios/docker/yarn-submit (checked 'origin/feature/replay-kube-job' branch).